<?xml version="1.0"?>
<launch>
    <node pkg="cv_camera" type="cv_camera_node" name="cv_camera">
        <param name="rate" value="10" />
    </node>

    <node pkg="gaze_tracking_ros" type="gaze_tracker.py" name="gaze_tracker">
        <remap from="image_raw" to="cv_camera/image_raw" />
    </node>
    
    <node pkg="gaze_tracking_ros" type="start_rqt.sh" name="rqt_gui" args="--perspective-file $(find gaze_tracking_ros)cfg/eye_tracking.perspective" required="true"/>
    
</launch>